content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,237 @@ The tables below show changes made to the installation options used to customize
-| `global.sds.udsPath` | `""` |  |
-| `global.sds.useTrustworthyJwt` | `false` |  |
-| `global.sds.useNormalJwt` | `false` |  |
-| `global.meshNetworks` | `{}` |  |
-| `global.enableHelmTest` | `false` | `Specifies whether helm test is enabled or not.This field is set to false by default, so 'helm template ...'will ignore the helm test yaml files when generating the template` |
+| `galley.nodeSelector` | `{}` |  |
+| `galley.tolerations` | `[]` |  |
+| `galley.podAntiAffinityLabelSelector` | `[]` |  |
+| `galley.podAntiAffinityTermLabelSelector` | `[]` |  |
 
 ### New `mixer` key/value pairs
 
 | Key | Default Value | Description |
 | --- | --- | --- |
-| `mixer.env.GODEBUG` | `gctrace=1` |  |
-| `mixer.env.GOMAXPROCS` | `"6"` | `max procs should be ceil(cpu limit + 1)` |
-| `mixer.policy.enabled` | `false` | `if policy is enabled, global.disablePolicyChecks has affect.` |
-| `mixer.policy.replicaCount` | `1` |  |
-| `mixer.policy.autoscaleEnabled` | `true` |  |
-| `mixer.policy.autoscaleMin` | `1` |  |
-| `mixer.policy.autoscaleMax` | `5` |  |
-| `mixer.policy.cpu.targetAverageUtilization` | `80` |  |
-| `mixer.telemetry.enabled` | `true` |  |
-| `mixer.telemetry.replicaCount` | `1` |  |
-| `mixer.telemetry.autoscaleEnabled` | `true` |  |
-| `mixer.telemetry.autoscaleMin` | `1` |  |
-| `mixer.telemetry.autoscaleMax` | `5` |  |
-| `mixer.telemetry.cpu.targetAverageUtilization` | `80` |  |
-| `mixer.telemetry.sessionAffinityEnabled` | `false` |  |
-| `mixer.telemetry.loadshedding.mode` | `enforce` | `disabled, logonly or enforce` |
-| `mixer.telemetry.loadshedding.latencyThreshold` | `100ms` | `based on measurements 100ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.` |
-| `mixer.telemetry.resources.requests.cpu` | `1000m` |  |
-| `mixer.telemetry.resources.requests.memory` | `1G` |  |
-| `mixer.telemetry.resources.limits.cpu` | `4800m` | `It is best to do horizontal scaling of mixer using moderate cpu allocation.We have experimentally found that these values work well.` |
-| `mixer.telemetry.resources.limits.memory` | `4G` |  |
-| `mixer.podAnnotations` | `{}` |  |
-| `mixer.nodeSelector` | `{}` |  |
-| `mixer.adapters.kubernetesenv.enabled` | `true` |  |
-| `mixer.adapters.stdio.enabled` | `false` |  |
-| `mixer.adapters.stdio.outputAsJson` | `true` |  |
-| `mixer.adapters.prometheus.enabled` | `true` |  |
-| `mixer.adapters.prometheus.metricsExpiryDuration` | `10m` |  |
-| `mixer.adapters.useAdapterCRDs` | `true` | `Setting this to false sets the useAdapterCRDs mixer startup argument to false` |
+| `mixer.telemetry.reportBatchMaxEntries` | `100` | `Set reportBatchMaxEntries to 0 to use the default batching behavior (i.e., every 100 requests). A positive value indicates the number of requests that are batched before telemetry data is sent to the mixer server` |
```

Comment:
I thought this change is only for master, not in 1.2...cc @huang195

Reply:
@linsun @huang195 I will run this against the 1.2 branch and can verify this again.